cli/common.go

```diff
@@ -62,7 +93,7 @@ func createRequest(dirName string) (controller.Request, error) {
 		req.Units = append(req.Units, controller.Unit{Name: name, Content: content})
 	}
 
-	sliceIDs, err := createSliceIDs(dirName)
+	sliceIDs, err := createSliceIDs(slices)
```

Comment:
do we maybe want to return the actual groupname from `createSliceIDs` too? Our `controller.Request` is missing that, although I think we should add it there.